lib/logstash/inputs/LogStashRecordProcessorFactory.rb

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+#
+#Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+require 'java'
+require_relative "LogStashRecordProcessor"
+begin
+  require 'jar-dependencies'
+  require_jar( 'com.amazonaws', 'amazon-kinesis-client', '1.6.0' )
+end
+
+module KCL
+  include_package "com.amazonaws.services.kinesis.clientlibrary.interfaces"
+end
+
+class LogStashRecordProcessorFactory
```

Comment:
The module thing commented before might also apply here, so the class name would be more friendly. what do you think?